utility-belt/src/test/java/io/confluent/admin/utils/EmbeddedZookeeperEnsemble.java

```diff
@@ -119,6 +122,7 @@ public void start() throws IOException {
       s.start();
     }
 
+    System.out.println("Checking ports " + hostPort);
```

Comment:
is this just leftover or is there a reason we're mixing slf4j logs and `System.out.println`?

Reply:
Yes, my bad.